airflow/www/views.py

```diff
@@ -131,10 +132,31 @@ def get_date_time_num_runs_dag_runs_form_data(request, session, dag):
     }
 
 
+def encode_ti(ti: models.TaskInstance) -> Optional[List]:
```

Comment:
Is this only used inside one view? If so it should probably be a nested function